examples/basic_operations/add_campaigns.rb

```diff
@@ -25,11 +25,12 @@ def add_campaigns(customer_id)
   # GoogleAdsClient will read a config file from
   # ENV['HOME']/google_ads_config.rb when called without parameters
   client = Google::Ads::GoogleAds::GoogleAdsClient.new
+  resource = Google::Ads::GoogleAds.resource
```

Comment:
Let's call this "resources" and also add a convenience access to it on the client.